src/bindings/js/node/lib/addon.ts

```diff
@@ -38,6 +38,7 @@ interface Core {
   readModelSync(modelBuffer: Uint8Array, weightsBuffer?: Uint8Array): Model;
   importModelSync(modelStream: Buffer, device: string): CompiledModel;
   getAvailableDevices(): string[];
+  getVersions(deviceName: string):{ deviceName: string, version: string, versionDescription: string }[];
```

Comment:
I propose to follow original format: 
```suggestion
  getVersions(deviceName: string): { 
    [deviceName: string]: { 
      buildNumber: string,
      description: string,
    },
  };
```

Reply:
Also, please check that your IDE is using and following project eslint rules